src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.vcxproj

```diff
@@ -77,7 +76,7 @@
       <DisableSpecificWarnings>4702;%(DisableSpecificWarnings)</DisableSpecificWarnings>
     </ClCompile>
     <Link>
-      <AdditionalDependencies>User32.lib;WindowsApp.lib;shell32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
I mean, wait, if this is using the cppwinrt pre props you don't even need the link block at all.

Reply:
:facepalm:

Reply:
wait no though I do?

Reply:
The cppwinrt preprops only was WindowsApp.lib...